libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -89,6 +89,26 @@ class AC_AttitudeControl {
     float get_accel_pitch_max() const { return _accel_pitch_max; }
     float get_accel_pitch_max_radss() const { return radians(_accel_pitch_max * 0.01f); }
 
+    // get and set target attitude quaternion
+    const Quaternion& get_attitude_target_quat() {return _attitude_target_quat; }
```

Comment:
these getters could be made const I suspect..

Reply:
oops, that was a non-const duplicate... fixing it